src/main/java/FileUtils.java

```diff
@@ -0,0 +1,36 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+
+public class FileUtils {
+
+    ArrayList<Integer> numbers = new ArrayList<>();
```

Comment:
Staraj się unikać dodawania pól, czyli tworzenia stanowości w klasach funkcyjnych. Lepiej w takich miejscach przekazywać wartość przez parametry metod. Przez zastosowanie takiego podejścia użytkownik Twojej klasy musi wykonać akcje w odpowiedniej kolejności (najpierw `saveNumbersFromFile`, a potem `printNumbersOccurrence`, bo inaczej się program wykrzaczy. Spróbuj to poprawić, żeby nie dało się wykonać akcji w błędnej kolejności.